clang/test/CIR/Lowering/brcond.c

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -fclangir-mem2reg -emit-llvm %s -o %t.ll
+
+#include <stdbool.h>
```

Comment:
We don't include outside headers. You can rename this file to `brcond.cpp` and remove the include.

Reply:
This test specifically is not reproducible in C++. I want this cast to be emitted `cir.cast(int_to_bool, %0 : !s32i), !cir.bool`, but it won't be emitted in C++) It seems using a CIR file for the test instead is a solution?

Reply:
In that case you need to use some mocked up version of `stdbool.h`. See `clang/test/Analysis/fields.c` as an example.